.github/workflows/ca-sequential-test.yml

```diff
@@ -1836,6 +1836,551 @@ jobs:
 
           diff expected actual
 
+      ####################################################################################################
+      # Enroll a cert with newLegacy
+      #
+      # This should work like the legacy .
+      - name: Switch to newLegacy
+        run: |
+          # switch cert request ID generator to RSNv3
+          BEGIN_SERIAL=$(docker exec pki pki-server ca-config-show dbs.beginSerialNumber)
+          docker exec pki pki-server ca-config-set dbs.beginSerialNumber 0x$BEGIN_SERIAL
+          END_SERIAL=$(docker exec pki pki-server ca-config-show dbs.endSerialNumber)
+          docker exec pki pki-server ca-config-set dbs.endSerialNumber 0x$END_SERIAL
+          docker exec pki pki-server ca-config-set dbs.request.id.generator newLegacy
+
+          docker exec pki pki-server ca-config-set dbs.cert.id.generator newLegacy
+
+          docker exec pki pki-server ca-range-update
```

Comment:
This command is actually meant to be run during installation only since it takes the `dbs.endSerialNumber` of the server being installed, increments it by 1, then stores it in the `nextRange` attribute which will be used by all other servers in the cluster. I suppose if the command is run on a server that doesn't have the latest range it might potentially create a conflicting `nextRange`.

Reply:
I was thinking that if no clones are present then it should not be a problem because the following range is not used yet. If there are clones a different approach should be used. If my assumption are not correct we have to find something different also for single CA setup. 
In my test it works if I set the `beginSerialNamber` in  the main CA higher then all the ranges available but it could create a big gap if increments are big. 
I am investigating other approach which avoid creating gaps

Reply:
I think we should find a solution that will work for clones, which will be the case for most production environments. If the solution works for clones it'll likely work for single machines too, but not the other way around.

Reply:
I have replaced this command with a ldapmodify. The value are retrieved from DS ranges but in running instance, if ranges are not created yet, these can be retrieved from CS.cfg without problems.
In case of clones, for the master CA this should work but for the clones, which may have sub-range we have to verify how to update.